modules/client-configuration/pages/autoinst-cdrom.adoc

```diff
@@ -0,0 +1,116 @@
+[[autoinst-cdrom]]
+= Autoinstallation via a CD-ROM
+
+For clients that are not yet registered to {productname}, and if PXE booting is not an option, a bootable CD-ROM can be used to install the system.
+
+One option to prepare such a CD-ROM is to use Cobbler.
```

Comment:
Replace prepare with finish. THIS IS A TEST.

Reply:
THIS IS A TEST. it works

Reply:
Original version accepted.